ansible/roles/mikegleasonjr.firewall/tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+
+- name: Install iptables
```

Comment:
Upstream says:

> **Requirements:**
> iptables (installed by default on all official Debian and RedHat distributions)

So in that context it seems to should move to bootstrap; I'll do that.